Firestore/core/src/firebase/firestore/model/document.h

```diff
@@ -49,15 +64,23 @@ class Document : public MaybeDocument {
   }
 
   bool has_local_mutations() const {
-    return has_local_mutations_;
+    return document_state_ == DocumentState::kLocalMutations;
+  }
+
+  bool has_committed_mutations() const {
+    return document_state_ == DocumentState::kCommittedMutations;
+  }
+
+  bool HasPendingWrites() const override {
```

Comment:
Question: this function uses PascalCase, while the other two very similar functions use snake_case. Can these be in the same style? (no strong preference which)